src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -856,8 +856,9 @@ public String getFeedbackSessionResultsSummaryInSectionAsCsv(
             String feedbackSessionName, String courseId, String userEmail,
             String section, String filterText, boolean isMissingResponsesShown, boolean isStatsShown)
             throws EntityDoesNotExistException, ExceedingRangeException {
+                      
+        long indicatedRange = section == null ? 8000 : -1;
```

Comment:
Suggest adding a new constant value for this, something like INSTRUCTOR_RESPONSE_LIMIT?